src/components/ColorSchemeToggle.vue

```diff
@@ -81,6 +81,10 @@ export default {
   @media print {
     display: none;
   }
+
+  @media (scripting: none) {
```

Comment:
Unfortunately, it looks like your suspicion is correct about this media feature not being widely supported enough, so we should probably look into an alternate approach for this. The global percentage of support for this is roughly about 8% according to [caniuse](https://caniuse.com/mdn-css_at-rules_media_scripting), which doesn't seem good enough.

Here's my current thinking on a possible alternate approach to solve this without that media query:

1. Update the current `<noscript>` element from the main index file (or add a new one) to include a reference to a new global CSS asset, which will hold global CSS styles that _only_ apply when JS is disabled.
```html
  <!-- this stylesheet is only included when JS is disabled -->
  <noscript><link rel="stylesheet" href="noscript.css" /></noscript>
```
2. Update that JS-disabled `noscript.css` file to include the following css rule:
```css
/* hide any element with the `data-js-required` attribute when JS is disabled */
*[data-js-required] {
  display: none;
}
```
3. For any component that you want to hide when JS is disabled, simply add the `data-js-required` attribute to the top level HTML element in its component.
```html
<template>
  <fieldset
    class="color-scheme-toggle"
    role="radiogroup"
    data-js-required
  >
```

I haven't tried any of this out, but it seems like it might be a fairly clean approach to doing this in a more compatible way.

There are other less clean ways like setting a default `display: none;` to things you want to hide without JS and then unhiding them in a mounted hook or something.

This media query would be the best solution once it's more compatible in the future though!

Reply:
I like that idea! There are a couple of things (like the breadcrumbs) where I want to do something other than hiding, so I might use the `no-js` class approach I mentioned in the intial comment to handle those, but the data attribute is a really cool idea too!

Reply:
For sure. You could use distinct attributes to handle different types of styling changes, but I'll let you decide which approach fits best.